client/src/components/Map/Map.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { GoogleMap, LoadScript } from '@react-google-maps/api';
+
+const containerStyle = {
+  width: 'auto',
+  height: '400px',
+};
+
+const center = {
+  lat: 50.429407,
+  lng: 30.591509,
+};
+
+function Map() {
+  return (
+    <LoadScript googleMapsApiKey="AIzaSyA5TINnTB_Yan0MjSGYVPfzgeZEOrMFLWA">
+      <GoogleMap
+        id="marker-example"
+        mapContainerStyle={containerStyle}
+        center={center}
+        zoom={16}
+        mapContainerClassName="mapContainerStyle"
```

Comment:
at least I suggest moving file with this class to `Map` folder instead of `ContactsPage` folder